```
make[3]: warning: jobserver unavailable: using -j1.  Add `+' to parent make rule.
```

Hmm, this is what my patch was supposed to fix.

```
../libssl.so: error: undefined reference to 'ERR_put_error'
../libssl.so: error: undefined reference to 'sk_dup'
../libssl.so: error: undefined reference to 'EVP_SignFinal'
```

Not sure what broke here.  Investigating…

--- a/python/servo/post_build_commands.py
+++ b/python/servo/post_build_commands.py
@@ -215,3 +215,26 @@ class PostBuildCommands(CommandBase):
         except subprocess.CalledProcessError as e:
             print("Packaging Android exited with return value %d" % e.returncode)
             return e.returncode
+
+    @Command('install',
+             description='Install Servo (currently, Android only)',
+             category='post-build')
+    @CommandArgument('--release', '-r', action='store_true',
+                     help='Package the release build')
+    @CommandArgument('--dev', '-d', action='store_true',
+                     help='Package the dev build')
+    def install(self, release=False, dev=False):
+        binary_path = self.get_binary_path(release, dev, android=True)
+        if not path.exists(binary_path):
+            # TODO: Run the build command automatically?
+            print("Servo build not found. Please run `./mach build` to compile Servo.")
+            return 1
+
+        apk_path = binary_path + ".apk"
+        if not path.exists(apk_path):
+            result = Registrar.dispatch("package", context=self.context, release=release, dev=dev)
+            if result is not 0:
+                return result
+
+        print(["adb", "install", "-r", apk_path])
+        return subprocess.call(["adb", "install", "-r", apk_path], env=self.build_env())

--- a/python/servo/post_build_commands.py
+++ b/python/servo/post_build_commands.py
@@ -194,10 +194,7 @@ class PostBuildCommands(CommandBase):
                      help='Package the release build')
     @CommandArgument('--dev', '-d', action='store_true',
                      help='Package the dev build')
-    @CommandArgument(
-        'params', nargs='...',
-        help="Command-line arguments to be passed through to Servo")
-    def package(self, params, release=False, dev=False, debug=False, debugger=None):
+    def package(self, release=False, dev=False, debug=False, debugger=None):
         env = self.build_env()
         binary_path = self.get_binary_path(release, dev, android=True)
 

